apps/dashboard/src/components/conditions-editor/operator-selector.tsx

```diff
@@ -0,0 +1,19 @@
+import { OperatorSelectorProps } from 'react-querybuilder';
+
+import { Select, SelectContent, SelectTrigger, SelectValue } from '@/components/primitives/select';
+import { cn } from '@/utils/ui';
+import { toSelectOptions } from '@/components/conditions-editor/select-option-utils';
+
+export const OperatorSelector = ({ disabled, value, options, handleOnChange }: OperatorSelectorProps) => {
```

Comment:
This component represents the operator field, in this example `=`.
![Screenshot 2025-01-14 at 12 30 06](https://github.com/user-attachments/assets/8b2be65e-b418-4d12-86b6-ec0f436b22cf)